tests/mock_data/calicoctl/mesh/static-routes/kubectl-input.yaml

```diff
@@ -0,0 +1,67 @@
+# A Service and corresponding endpoints object.
```

Comment:
New test which does basic validation of the "dynamic" aspect of route programming.

Creates a svc/endpoints and checks the right clusterIPs show up.

Reply:
Nice!